plugins/kosync.koplugin/main.lua

```diff
@@ -25,12 +25,19 @@ local KOSync = InputContainer:new{
     title = _("Register/login to KOReader server"),
 }
 
+-- Creates a unique (hopefully) device id to identify this device. This function will only be
+-- called during the very first time KOSync has been initialized.
+local function createDeviceId()
+    return tostring(math.random(100,999))
```

Comment:
That's a good point. I suggest we should at least increase the entropy then. 0.1% feels like a percentage to me. How about encoding 2 random bytes with hex?